### DISCUSSION TITLE
Marketing is a little weird

### DISCUSSION THREAD

**Discussion by IcedQuinn** [google/zx#384] [General]
I was shown this project elsewhere and initially criticized it because it seems to want to replace regular shell scripts from reading the text. After a bit of discussion and re-reading I kind of get its place now (ex. in similar veins of pydoit, fabric or rake, where the script is interacting with a lot of APIs more than just trudging some files around like shell is for.)

Will have to think of a description and some examples that more show off particularly when you would want to use this above slapping a shell script in place. Something peculiar like running a command which returns JSON and then iterating the JSON to run other batch commands. (the current examples don't really demonstrate why i would want to bring in node+npm+zx if i don't already use node for example.) I know some people have some javascript faceplates for building up calls to ffmpeg as an example.

**Comment by antonmedv** [comment]
Obviously, zx targets nodejs devs. You can take a look on how people are using it: https://github.com/google/zx/network/dependents
